documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -78,3 +78,39 @@ Using StrimziPodSets instead of StatefulSets provides more control over the func
 To enable the `UseStrimziPodSets` feature gate, specify `+UseStrimziPodSets` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
 
 IMPORTANT: The `UseStrimziPodSets` feature gate must be disabled when downgrading to Strimzi 0.27 and earlier versions.
+
+[id='ref-operator-use-kraft-feature-gate-{context}']
+== (Preview) UseKRaft feature gate
+
+The `UseKRaft` feature gate has a default state of _disabled_.
+
+The `UseKRaft` feature gate deploys the Kafka cluster in the KRaft (Kafka Raft metadata) mode without ZooKeeper.
+This feature gate is currently intended only for development and testing.
+
+IMPORTANT: **The KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
+
+When the `UseKRaft` feature gate is enabled, the Kafka cluster is deployed without ZooKeeper.
+The `.spec.zookeeper` properties in the Kafka custom resource will be ignored, but still need to be present.
```

Comment:
I would highlight more this in bold, at least the part "still need to be present".